homeworks/07-js-basics-2/index.js

```diff
@@ -48,20 +45,19 @@ function getEvenNumbers(numbersArray) {
 // 5 https://www.codewars.com/kata/5a090c4e697598d0b9000004
 
 function solve(arr) {
-  let cpyOne = [...arr];
-  let cpyTwo = [...arr];
+  let cpyArr = [...arr];
 
-  cpyOne.sort((a, b) => (a < b ? 1 : -1));
-  cpyTwo.sort((a, b) => (a > b ? 1 : -1));
+  arr.sort((a, b) => (a < b ? 1 : -1));
```

Comment:
Now you have mutated incoming array instead of copying - it is even worst.
The point was to work with only 1 sorted copy of array and empty result array for filling (you have max value from the beginning and min value at the end - that is enough for task solving).